test/document.test.js

```diff
@@ -1011,13 +1011,13 @@ describe('document', function() {
   });
 
   describe('Errors', function() {
-    it('MongooseErrors should be instances of Error (gh-209)', function(done) {
+    it('MongooseError should be instances of Error (gh-209)', function(done) {
```

Comment:
The test name as written was grammatically correct, but now it isn't. Both 'MongooseError' and 'instance' should be both singular or both plural.